test/integration/chef_ci_cookbook_template/inspec/chef_ci_cookbook_template_spec.rb

```diff
@@ -1,11 +1,7 @@
-require 'serverspec'
-
-set :backend, :exec
-
 describe package('chefdk') do
   it { should be_installed }
 end
 
 describe command('chef --version') do
-  its(:stdout) { should match(/2\.5\.3/) }
+  its('stdout') { should match(/2\.5\.3/) }
```

Comment:
This test fails. It looks like when you rebased, you didn't update the version to ``2.6.3``.